mne/io/brainvision/brainvision.py

```diff
@@ -55,7 +55,7 @@ class RawBrainVision(_BaseRaw):
         triggers will be ignored. Default is 0 for backwards compatibility, but
         typically another value or None will be necessary.
     event_id : dict | None
-        The id of the event to consider. If None (default),
+        The id of special events to consider. If None (default),
```

Comment:
event_id is always checked first, then the regex is checked. So the event_id is truly only for special events (like Sync etc). Do you think that's not clear from the doc?